cmd/root.go

```diff
@@ -274,11 +281,13 @@ func JsonnetVM(cmd *cobra.Command) (*jsonnet.VM, error) {
 				if len(kv) != 2 {
 					return nil, fmt.Errorf("Failed to parse %s: missing '=' in %s", spec.flagName, entry)
 				}
-				v, err := ioutil.ReadFile(kv[1])
-				if err != nil {
-					return nil, err
+				var imp string
+				if spec.isCode {
+					imp = "import"
+				} else {
+					imp = "importstr"
 				}
-				spec.inject(kv[0], string(v))
+				spec.inject(kv[0], fmt.Sprintf("%s @'%s'", imp, strings.ReplaceAll(kv[1], "'", ",,")))
```

Comment:
I think this replaces `'` with `,,` ?!
```suggestion
				spec.inject(kv[0], fmt.Sprintf("%s @'%s'", imp, strings.ReplaceAll(kv[1], "'", "''")))
```

Reply:
Damned DVORAK keyboard! I'm glad you figured out what I meant.